src/Razor/test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic_DesignTime.ir.txt

```diff
@@ -42,12 +43,16 @@ Document -
                     IntermediateToken - (54:2,5 [29] Basic.cshtml) - CSharp - string.Format("{0}", "Hello")
                 HtmlContent - (83:2,34 [10] Basic.cshtml)
                     IntermediateToken - (83:2,34 [2] Basic.cshtml) - Html - \n
-                    IntermediateToken - (85:3,0 [6] Basic.cshtml) - Html - </div>
+                    IntermediateToken - (85:3,0 [1] Basic.cshtml) - Html - <
+                    IntermediateToken - (86:3,1 [1] Basic.cshtml) - Html - /
+                    IntermediateToken - (87:3,2 [3] Basic.cshtml) - Html - div
+                    IntermediateToken - (90:3,5 [1] Basic.cshtml) - Html - >
```

Comment:
hmmm

Reply:
did you expect this? I was hoping we could do more than one character per node.

Reply:
Yes, I was definitely expecting this. This still all under one node(`HtmlContentIntermediateNode`) just split into tokens. We already spoke about this offline when I made a related change earlier https://github.com/aspnet/AspNetCore/pull/4326#discussion_r237690311. I am actually surprised that you weren't expecting this.

Reply:
> why is this a good change for the IR?

Just adds more clarity. I don't think it's a bad change though. The alternative is to do extra steps to merge the tokens together at the IR phase. This will also not be straightforward because we'll need to merge in different ways depending on whether attributes are present or not etc.

Reply:
That seems **SUPER** appropriate for the IR lowering phase. Remember, we want the structure to change between phases, that's the point. The IR is not a verbatim translation of the syntax tree.